app/controllers/web_controller.rb

```diff
@@ -147,6 +147,8 @@ def set_logging_attributes
         CurrentLoggingAttributes.user_organisation_slug = current_user.organisation&.slug
       end
     end
+
+    CurrentLoggingAttributes.params = params.except(:controller, :action, :format, :id).to_unsafe_h
```

Comment:
Should we also be excluding `authenticity_token`?

Reply:
Huh, I hadn't seen that in `params` when running on my machine, where did it come up?

Reply:
Oh I see it now, I had only been paying attention to what was logged in the tests, but that doesn't include the CSRF token. Good catch, I'll fix this.